app/models/manageiq/providers/vmware/infra_manager/vm/reconfigure.rb

```diff
@@ -52,6 +52,117 @@ def build_config_spec(task_options)
       end
       set_spec_option(vmcs, :memoryMB, task_options[:vm_memory],      :to_i)
       set_spec_option(vmcs, :numCPUs,  task_options[:number_of_cpus], :to_i)
+
+      if task_options[:disk_remove] || task_options[:disk_add]
+        with_provider_object do |vim_obj|
+          task_options[:disk_remove].each { |d| remove_disk_config_spec(vim_obj, vmcs, d) } if task_options[:disk_remove]
+          add_disks(vim_obj, vmcs, task_options[:disk_add]) if task_options[:disk_add]
+        end
+      end
+    end
+  end
+
+  def add_disks(vim_obj, vmcs, disks)
+    ck, un = vim_obj.send(:getScsiCandU)
```

Comment:
what are `ck` and `un` here?  can you use a more descriptive name?

I was wondering if it was somehow tied to the `C` and `U` in `getScsiCandU`, but I couldn't figure out what they would mean.